example/app/config.yml

```diff
@@ -143,6 +143,13 @@ aimee-test:
     instanceProfile: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:instance-profile/{{prefix}}-ecs'
     distributionRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/{{prefix}}-distribution-api-lambda'
 
+  communityConfiguration:
+    ApiTokenDefault:
```

Comment:
You can of course do whatever you want here, but I was wondering if it might be a better user experience to have some kind of `authorization` key, which you could set to `Google` and then internally set it to use the correct handler. That way you don't have to expose `ApiTokenDefault` which probably doesn't make sense to an end user. Also we have multiple versions of the Api (v1 and default for now), so I don't know if this would be the best way to configure this long term.